### DISCUSSION TITLE
Convert from Geographic to Local Cartesian ENU

### DISCUSSION THREAD

**Discussion by henningvz** [boostorg/geometry#1117] [Q&A]
Hi,
I wonder if it is possible with bg to define a transformation from geographic coordinates (lat,lon,h in WGS84) to a local cartesian coordinates ENU (x,y,z) given a geographic reference point? Currently I use the corresponding function from Karney's Geographiclib, but i would like to switch completely to bg. Unfortunately i have not found any documentation on this (yet).  Any hints or comments?
Thank in advance,
Henning

**Comment by vissarion** [comment]
Hi, projections are implemented in BG based on the implementation of `proj4`, see https://github.com/boostorg/geometry/tree/develop/include/boost/geometry/srs/projections/proj 
Indeed the implementation is not documented yet, but you will find many examples of how to use the interface in unit tests of BG. See for example https://github.com/boostorg/geometry/blob/develop/test/srs/projections.cpp
